real_time/arduino_src/radio_buggy_mega/fingerprint.py

```diff
@@ -1,7 +1,7 @@
-""" fingerprint.py: 
```

Comment:
It adds unnecessary diffs and makes the "blame" command useless on Github though